### DISCUSSION TITLE
Can notification messages include release notes ?

### DISCUSSION THREAD

**Discussion by schklom** [containrrr/watchtower#967] [Q&A] [answered]
**Is your feature request related to a problem? Please describe.**
Finding the release notes for containers updating is not fun.

**Describe the solution you'd like**
Adding a `label` to containers that would contain a link to their GitHub's release page, or their GitHub's repo name.
Using that, I assume it should be manageable to retrieve the release notes? I might be able to do it in shell (I see that some of the code is in shell), but I have 0 knowledge about Go.

**Describe alternatives you've considered**
Watch the release pages and notify changes using RSS feeds. But I would rather use the service that updates containers to tell me after an update what the change was.

An extension of this could be: Watchtower sees a container update, notifies the user about the changes (email or whatever), and updates next time it checks for updates. This setup would give the user some time to prevent an update if needed.

And for a more general (but a bit more complicated) extension: add a parameter for the delay, something like an integer for the number of hours before actually updating, or a number of Watchtower runs to delay updates (3 would mean delay updates for 3 Watchtower process runs).

**Comment by simonaronsson** [comment, marked as answer]
You can probably already achieve this using the `post-update` lifecycle hook. Have a look at this part of the documentation: https://containrrr.dev/watchtower/lifecycle-hooks/

**Reply by schklom** [reply]
Cool!
I didn't know about this, thanks 👍

**Reply by milindpatel63** [reply]
Can you provide an example of how to achieve this using a post-update hook?

**Reply by walkertr0n** [reply]
I would also appreciate an example of this! This idea sounds amazing tbh:

> Watchtower sees a container update, notifies the user about the changes (email or whatever), and updates next time it checks for updates. This setup would give the user some time to prevent an update if needed.
